.github/workflows/ingest-to-phylogenetic.yaml

```diff
@@ -39,11 +39,49 @@ jobs:
         ingest/logs/
         ingest/.snakemake/log/
 
-  # TKTK check if ingest results include new data
-  # potentially use actions/cache to store Metadata.sha256sum of S3 files
+  # Check if ingest results include new data by checking for the cache
```

Comment:
This relies on the ordering of metadata & sequences to be identical across runs, right? I don't think any of our ingest steps sort the data, and any (default, multi-threaded) call to `nextclade run` will change the ordering. I also wouldn't think the NCBI data is guaranteed to be in the same order across different `dataset` / `entrez` runs, but maybe?

Reply:
> This relies on the ordering of metadata & sequences to be identical across runs, right? 

Yes! We do not do any explicit sorting, so change in order from upstream data can result in triggering the phylogenetic workflow. However, we've been using this method for ncov-ingest and mpox with `dataset` and it's worked pretty well. 

> any (default, multi-threaded) call to nextclade run will change the ordering. 

Nextclade can potentially change ordering of the aligned.fasta, but not the sequences.fasta. I don't think it will have an influence on metadata, since usually the joining of the metadata and nextclade data outputs based on metadata order. If we are worried about Nextclade order we can use the `--in-order` flag to guarantee output order is the same as input order. Although, this repo currently does not include `nextclade run` in the ingest workflow.